_summaries/ShiviAgarwal_Adadelta.md

```diff
@@ -0,0 +1,54 @@
+ADADELTA: AN ADAPTIVE LEARNING RATE METHOD, Zeiler1; 2012 - Summary
+
+author:	Shivi Agarwal
+
+score:	 / 10
+
+ADADELTA 
+
+
+•	What is the core idea?
+The paper introduces ADADELTA a learning rate used in gradient descendent algorithm for optimization of neural network.
+ADADELTA is computed on a per-dimension basis using only first order information.
+ADADELTA is better to other learning rates because it is robust to large gradients, noise and architecture choice and it requires minimal calculation over gradient descent.
```

Comment:
What, if any, are the drawbacks? Based on this statement and the following statement about how it addresses the two main issues in ADAGRAD it seems like ADADELTA is really good. Are there any scenarios or tasks where you might expect it to struggle ?